models/budget.js

```diff
@@ -11,11 +11,11 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Budget.belongsTo(models.User);
     }
   };
   Budget.init({
     dollarAmount: DataTypes.DECIMAL,
-    UserId: DataTypes.INTEGER,
```

Comment:
With the association set up above, we can remove the UserId column from the rest of the attributes...